installCatalog.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# use the command line interface to install standard actions deployed
+# automatically
+#
+# To run this command
+# ./installCatalog.sh  <AUTH> <APIHOST> <WSK_CLI>
+# AUTH and APIHOST are found in $HOME/.wskprops
+# WSK_CLI="$OPENWHISK_HOME/bin/wsk"
+
+set -e
+set -x
+
+if [ $# -eq 0 ]
+then
+echo "Usage: ./installCatalog.sh <authkey> <apihost> <pathtowskcli>"
+fi
+
+AUTH="$1"
+APIHOST="$2"
+WSK_CLI="$3"
+
+# If the auth key file exists, read the key in the file. Otherwise, take the
+# first argument as the key itself.
+if [ -f "$AUTH" ]; then
+    AUTH=`cat $AUTH`
+fi
+
+PACKAGE_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+export WSK_CONFIG_FILE= # override local property file to avoid namespace clashes
+
+echo Installing the kafka package and feed action.
+
+$WSK_CLI -i --apihost "$APIHOST" package update messaging \
+    --auth "$AUTH" \
+    --shared yes \
+    -p endpoint "$APIHOST" \
+    -p package_endpoint $KAFKA_PROVIDER_ENDPOINT
+
+$WSK_CLI -i --apihost "$APIHOST" action update messaging/kafkaFeed "$PACKAGE_HOME/action/kafkaFeed.js" \
+    --auth "$AUTH" \
+    --shared yes \
+    -a description 'Feed to listen to Kafka messages' \
+    -a parameters '[ {"name":"brokers", "required":true, "description": "Array of Kafka brokers"}, {"name":"topic", "required":true, "description": "Topic to subscribe to"}, {"name":"isJSONData", "required":false, "description": "Attempt to parse message content as JSON"}, {"name":"endpoint", "required":true, "description": "Hostname and port of OpenWhisk deployment"}]' \
```

Comment:
I see that Cloudant and Alarms do not document this property in annotations. @jasonpet could you weigh in on this?